pkg/lockfile/lockfile_unix.go

```diff
@@ -35,7 +35,7 @@ type lockfile struct {
 // necessary.
 func openLock(path string, ro bool) (int, error) {
 	if ro {
-		return unix.Open(path, os.O_RDONLY, 0)
+		return unix.Open(path, os.O_RDONLY|os.O_CREATE, 0)
```

Comment:
@nalind, do you have any concerns with that change?

Reply:
[The main reason behind that change is to be able to initially create the lock if read-only.]

Reply:
Yeah, this is fine (didn't know that O_RDONLY | O_CREAT was a thing, so that's neat).

Reply:
Awesome, thanks for the clarification.